js/modules/k6/execution/execution.go

```diff
@@ -170,6 +172,31 @@ func (mi *ModuleInstance) newInstanceInfo() (*goja.Object, error) {
 	return newInfoObj(rt, ti)
 }
 
+// newTestInfo returns a goja.Object with property accessors to retrieve
+// information and control execution of the overall test run.
+func (mi *ModuleInstance) newTestInfo() (*goja.Object, error) {
+	ctx := mi.GetContext()
+	rt := common.GetRuntime(ctx)
+	if rt == nil {
+		return nil, errors.New("goja runtime is nil in context")
```

Comment:
`exec.test.abort()` does work in the init context, see `TestInitContextForbidden/abortTest` (maybe the wrong parent test for this). It will just exit with code 255 instead of 107 right now. Should we fix that here or in a separate PR?

Reply:
Why does this check exist? I am pretty sure there is no place in which this is possible?

Reply:
@MStoykov I suppose extra precaution against a panic :smile: But if it's not possible, I removed it in fb517cef.